unit_test/hail_search/persistentvolume.yaml

```diff
@@ -18,18 +18,24 @@ volumeMounts: |-
     mountPath: {{ .Values.environment.IN_MEMORY_DIR }}
 
 initContainers: |-
-  {{- range $path, $version := index $.Values.global "hail-search" "datasetVersions" -}}
-  {{ if $version }}
-  {{- $localPath := $path | replace "SV" "SV_WGS" | replace "GCNV" "SV_WES" -}}
-  - name: sync-annotations-{{ $path | replace "/" "-" | replace "_" "-" | lower}}
+  - name: sync-annotations
     image: busybox
-    command: ['/bin/sh', '-c', 'mkdir -p {{ $.Values.environment.IN_MEMORY_DIR }}/{{ $localPath }} && cp -r {{ $.Values.environment.HAIL_SEARCH_DATA_DIR }}/{{ $localPath }}/annotations.ht {{ $.Values.environment.IN_MEMORY_DIR }}/{{ $localPath }}/annotations.ht']
+    command: [
+      '/bin/sh', 
+      '-c',
+      'mkdir -p {{ $.Values.environment.IN_MEMORY_DIR }}',
+      '&&',
+      'rsync',
```

Comment:
Without the loop over the `datasetVersions`, we can be much more efficient with the `rsync`, reducing the number of init containers and the pod startup time.